src/js/views/loading_view.js.coffee

```diff
@@ -0,0 +1,13 @@
+define [
```

Comment:
I'd like us to drop the 'js.coffee' extension in lieu of just 'coffee'. I think there are some .js.coffee files, but I think .coffee is more concise and would prefer that we use it.